.github/workflows/docker-indexer-grpc-test.yaml

```diff
@@ -36,23 +36,17 @@ jobs:
           AWS_DOCKER_ARTIFACT_REPO: ${{ secrets.AWS_DOCKER_ARTIFACT_REPO }}
           GIT_CREDENTIALS: ${{ secrets.GIT_CREDENTIALS }}
 
-      - name: Run validator-testnet docker compose
-        run: docker-compose up -d
+      - name: Install grpcurl
```

Comment:
This should probably be a composable action which we can share

Reply:
good call